apps/src/applab/designMode.js

```diff
@@ -613,12 +613,13 @@ designMode.onDuplicate = function(element, event) {
 
 var batchChangeId = 1;
 
-designMode.changeThemeForCurrentScreen = function(prevThemeValue, themeValue) {
-  const currentScreen = $(
-    elementUtils.getPrefixedElementById(
-      getStore().getState().screens.currentScreenId
-    )
-  );
+designMode.changeThemeForCurrentScreen = function(screenElement, themeValue) {
```

Comment:
It might make sense to change the name of this method to `changeThemeForScreen` since with the updated signature it may not always apply to the current screen.